proto/acl_rule.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package dash.acl_rule;
+
+import "types.proto";
+
+enum Action {
+    ACTION_DENY = 0;
+    ACTION_PERMIT = 1;
+}
+
+message AclRule {
+    // priority of the rule, lower the value, higher the priority
+    uint32 priority = 1;
+    // allow/deny
+    acl_rule.Action action = 2;
+    // true/false   ; if true, stop processing further rules
+    bool terminating = 3;
+    // Optional
+    // Protocol list. E.g. 6-tcp, 17-udp; if not provided, match on all protocols
+    repeated uint32 protocol = 4;
+    // Optional
+    // list of source tag name, if not provided, match on ANY tag or NO tag.
+    repeated string src_tag = 9;
```

Comment:
should be integer, i think.

Reply:
In the client view, the tag is a name string, like xxx_Backend, xxx_ResourceManager.
But there is a TagId, like ab6b2357-624c-4a39-ad4e-000000000000, to each TAG group. What do you think to use the GUID type?